src/analysis/classic/paladin/protection/modules/features/Buffs.tsx

```diff
@@ -0,0 +1,27 @@
+import CoreAuras from 'parser/core/modules/Auras';
+import SPELLS from 'common/SPELLS/classic';
+import BLOODLUST_BUFFS from 'game/BLOODLUST_BUFFS';
+import ITEM_BUFFS from 'game/classic/ITEM_BUFFS';
+
+class Buffs extends CoreAuras {
+  // A list of Buffs (on the current player) to highlight on the Timeline
+  auras() {
+    return [
+      {
+        spellId: SPELLS.PAIN_SUPPRESSION.id,
+        timelineHighlight: true,
+      },
```

Comment:
I don't remember WotLK era well enough, were there any other externals worth noting, or was it really just Pain Sup?

Reply:
There might be more but this is a stub. Hopefully a person who plays Prot Paladin can pick it up.

Reply:
Sure, works for me 👍